src/models/__tests__/Project.spec.ts

```diff
@@ -15,6 +15,38 @@ async function expectFieldPresence(field: keyof ProjectData) {
     expect(errors[field] !== undefined && (errors[field] || []).length > 0).toBeTruthy();
 }
 
+async function expectNumberAwardNumber(field: keyof ProjectData) {
```

Comment:
I abstracted `expectFieldPresence` because it was used with different fields, here it's a concrete field, it can be moved to the test itself, with explicit field name.